en/docs/try-out/account-information-service-flow.md

```diff
@@ -0,0 +1,896 @@
+user that has a subscriber role.
+
+3. The page displays the data requested by the TPP for that particular consent along with its accounts and permissions. 
+      - In a Bank offered consent scenario, the PSU is allowed to choose the accounts for the requested permissions. 
+      - The consent page will slightly differ based on the different consent types. 
+
+    ??? tip "Click here to see sample consent pages for different consent types"
+       
+          **Dedicated accounts consent**
+           ![dedicated_accounts_consent](../assets/img/try-out/account-flow/dedicated-accounts-consent.png)
+    
+          **Bank offered consent**
+          ![bank_offered_consent](../assets/img/try-out/account-flow/bank-offered-consent.png)
+      
+          **Account list of available accounts consent**
+          ![account_list_of_available_accounts_consent](../assets/img/try-out/account-flow/available-accounts-list-consent.png)
+    
+          **Global consent**
+          ![global_consents](../assets/img/try-out/account-flow/global-consent.png)
+
+4. Upon providing consent, an authorization code is generated on the web page of the `redirect_uri`. See the sample 
+given below:
+
+    ```
+    https://www.wso2.com/?code=5b771c49-dfe3-33f6-abe1-18f0426b8734&state=768ff3be-964c-41d9-9022-848ba379e1bf
+    ```
+
+    The authorization code from the above URL is in the code parameter (code=`5b771c49-dfe3-33f6-abe1-18f0426b8734`).
+
+### Generating user access token
+
+In this section, you will be generating an access token using the authorization code generated in the section [above](#authorizing-a-consent).
+
+1. Generate the client assertion by signing the following JSON payload using supported algorithms. 
+
+    !!! note
+        If you have configured the [OB certificates](https://openbanking.atlassian.net/wiki/spaces/DZ/pages/252018873/OB+Root+and+Issuing+Certificates+for+Sandbox), 
+        download the certificate and keys attached [here](../../assets/attachments/Certificates.zip), and use them for signing and transport layer security testing purposes.
+
+    ``` tab="Format"
+    Format:
+    {
+    "alg": "<The algorithm used for signing.>",
+    "kid": "<The thumbprint of the certificate.>",
+    "typ": "JWT"
+    }
+     
+    {
+    "iss": "<This is the issuer of the token. For example, client ID of your application>",
+    "sub": "<This is the subject identifier of the issuer. For example, client ID of your application>",
+    "exp": <This is the epoch time of the token expiration date/time>,
+    "iat": <This is the epoch time of the token issuance date/time>,
+    "jti": "<This is an incremental unique value>",
+    "aud": "<This is the audience that the ID token is intended for. For example, https://<IS_HOST>:9446/oauth2/token>"
+    }
+     
+    <signature: For DCR, the client assertion is signed by the private key of the signing certificate. Otherwise, the private signature of the application certificate is used.>
```

Comment:
Use the sample provided in the google doc here because there is no DCR for Berlin

Reply:
Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0